src/include/platform/PlatformManager.h

```diff
@@ -233,11 +235,32 @@ inline void PlatformManager::RunEventLoop()
     static_cast<ImplClass *>(this)->_RunEventLoop();
 }
 
+/**
+ * @brief
+ *  Starts the stack on its own thread/task with an associated event queue
+ *  to dispatch and handle events posted to that task.
+ *
+ *  Thread Safe: YES
```

Comment:
nit: maybe just "Thread safe" is sufficient without the key/value pair as we do not seem to use that anywhere else.
Can use "NOT thread safe" for the inverse.

Reply:
Do we maybe even need some completely different verbiage?  These interfaces are more than just thread safe: they are have call chains that acquire the platform lock.  This means the must _not_ be called from within the event loop itself.

Thread safe makes it sound like you can call these from any context, and that's not true at all.

Reply:
The connotation I want to describe to the reader is that it's externally thread synchronized, but also that the platform lock _might_ be held and **should not** be grabbed within this call.

Reply:
Suggested text:

```
 *  This is thread-safe.
 *  This is *NOT SAFE* to call from within the CHIP event loop since it can grab the stack lock.
```